src/components/MessageList/MessageList.tsx

```diff
@@ -142,25 +298,29 @@ const MessageList = (props) => {
         <DefaultMessage
           actionSheetStyles={actionSheetStyles}
           AttachmentFileIcon={AttachmentFileIcon}
-          groupStyles={message.groupStyles}
+          groupStyles={message.groupStyles as GroupType[]}
           lastReceivedId={lastReceivedId === message.id ? lastReceivedId : null}
           Message={Message}
           message={message}
           messageActions={messageActions}
           onThreadSelect={onThreadSelect}
-          readBy={message.readBy || []}
+          readBy={(message.readBy as UserResponse<Us>[]) || []}
```

Comment:
same as above comment, need to get on same page here

Reply:
same response as above